### PR TITLE
(blog) Ingestion

### PR DESCRIPTION
**Pull request by anna-geller**

### CODE REVIEW

File: content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+::alert{type="info"}
+To see how you can orchestrate Airbyte connectors with [kestra](https://github.com/kestra-io/kestra), check our [Airbyte blueprints](https://demo.kestra.io/ui/blueprints/community?page=1&q=airbyte), and [the following Airbyte documentation page](https://docs.airbyte.com/operator-guides/using-kestra-plugin/).
+::
+
+![airbyte_kestra.gif](/blogs/2023-10-09-why-ingestion-will-never-be-solved/airbyte_kestra.gif)
+
+Another distinction compared to [Singer](https://www.singer.io/) is that Airbyte is primarily designed to be used from the UI, not via code. This brings us to [CloudQuery](https://www.cloudquery.io/) and [dltHub](https://dlthub.com/) which are code-centric and intended to be used by developers.
```
Comment (tchiotludo) [inline content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md:125]: Maybe add the terraform provider that fill the gap ?

File: content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+Especially after Stitch was [acquired](https://www.talend.com/about-us/press-releases/talend-acquires-stitch-a-leader-in-self-service-cloud-data-integration/) by Talend, there was no central entity governing the quality of connectors, leading to eroding trust in the data community.
+
+### Open-source connector ecosystems
+
+[Airbyte](https://airbyte.com/), [CloudQuery](https://www.cloudquery.io/) and [dltHub]([dltHub](https://dlthub.com/)) learned from the early successes and failures of [Singer](https://www.singer.io/). Instead of building a standard, all of them implemented their own **framework** and **platform** for building connectors using a pluggable system. This means that the open-source community can contribute to the connectors, and the platform vendor can ensure that those contributions are maintained and kept up-to-date.
```
Comment (tchiotludo) [inline content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md:90]: > failures of singer 
is maybe hard word ? even more that we have the plugin in Kestra

File: content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+One of the main selling points of dlt is that it's a lightweight Python-based framework, allowing you to create new connectors with automatic schema inference and schema evolution. You can transform your data along the way e.g. to add pseudomization step before loading data to the target destination. And there are custom decorators to configure incremental and merge loads in Python.
+
+How does dltHub monetize? dlt plans to offer an [integration hub](https://www.cloudquery.io/register-for-cloud) which might resemble a connector marketplace and a managed service for data integration. It seems similar to CloudQuery's offering, but it's too early to tell how the two products will compare commercially.
```
Comment (tchiotludo) [inline content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md:142]: wrong link

File: content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md (hunk 2 above)
Comment (anna-geller) [reply content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md:90]: it says: successes and failures :)

File: content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md (hunk 3 above)
Comment (anna-geller) [reply content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md:142]: I switched them, need a link for both thanks

File: content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md (hunk 1 above)
Comment (anna-geller) [reply content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md]: ty, added

File: content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md (hunk 2 above)
Comment (anna-geller) [reply content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md]: I cut this now

**Pull request merged**